### PR TITLE
feat: add nvim-dap-ui extension

### PR DESCRIPTION
**Pull request by JavaHello**
Add [nvim-dap-ui](https://github.com/rcarriga/nvim-dap-ui) extension

### CODE REVIEW

**Comment by shadmansaleh** [comment]
You need to mention the extension in README.md with other extensions

**Comment by JavaHello** [comment]
> You need to mention the extension in README.md with other extensions

OK, I added the Available extensions in README.md

File: README.md [hunk 1]
```diff
@@ -761,6 +761,7 @@ extensions = {'quickfix'}
 - quickfix
 - symbols-outline
 - toggleterm
+- nvim-dap-ui
```
Comment (shadmansaleh) [inline README.md:764]: nitpick: the list is sorted by name.

**Comment by shadmansaleh** [comment]
I am not using dap-ui can you give a screenshot with this extension being used ?

**Comment by JavaHello** [comment]
> I am not using dap-ui can you give a screenshot with this extension being used ?
dap-ui-debug

File: README.md (hunk 1 above)
Comment (JavaHello) [reply README.md]: ok, i used the sort command to sort

File: lua/lualine/extensions/nvim-dap-ui.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
+-- MIT license, see LICENSE for more details.
+-- Extension for nvim-dap-ui
+local M = {}
+
+M.sections = {
+  lualine_a = { 'filename' }
```
Comment (shadmansaleh) [inline lua/lualine/extensions/nvim-dap-ui.lua:6]: Set `file_status` option to false then [-] won't appear . Also it seems after last commit there are merge conflicts.

Comment (JavaHello) [reply lua/lualine/extensions/nvim-dap-ui.lua:6]: ok thanks for the suggestion, i added the file_status option

**Comment by shadmansaleh** [comment]
lgtm, I think we can merge this now.

**Pull request merged**